doc/md/fundamentals/1-basic-syntax/9-operators.md

```diff
@@ -162,7 +162,7 @@ let result = 5 |> double(_) |> increment(_); // (5 * 2) + 1 = 11
 
 Option blocks use the syntax `do ? <block>` to handle optional values of type `?T` without needing nested switch statements. It produces a value of type `?T`, when `<block>` has type `T` and, importantly, introduces the possibility of a break from `<block>.`
 
-Within a `do ? <block>`, the `null` break `<exp> !` tests whether the result of the expression, `<exp>` of unrelated option type, `?U`, is `null`. 
+Within a `do ? <block>`, the `null` break `<exp> !` tests whether the result of the expression, `<exp>` of unrelated option type, `?U`, is `null`.
 
 If the result is `null`, control immediately exits the `do ? <block>` with value `null`. Otherwise, the result of `<exp>` must be an option value `?v`, and evaluation of `<exp> !` proceeds with its contents, `v` of type `U`.
 
```

Comment:
The following eval example is actually good (especially if the whole sections is moved to a more advance section).

I would make the example self-contained and executable (not `no-repl`), by defining the `Exp` type in the example, something like:
```
 type Exp = {#Lit : Nat; #Div : (Exp, Exp); #If : (Exp, Exp, Exp)};
 func eval ....
```